document/CoreFormat.mdk

```diff
@@ -1120,11 +1185,78 @@ C++: `Namespace::Class::Method(int, double) const &&`
 C#: `Namespace1.Namespace2.Class.Method(System.String, int[])`
 ~
 
+If the `run.logicalLocations` property (&sect;[#run-logicalLocations]) is present,
+the value of the `fullyQualifiedLogicalName` property should be equal to the name of one of the properties on the `run.logicalLocations` object,
+with one exception, described in &sect;[#location-logicalLocationKey].
+
 ~ IsoNote
 The intent of this property is to allow a result log viewer to display the logical location
```

Comment:
Yes. I now have bits of this in two different locations. Last change before I merge is to figure out where to combine this information and put it in one place.